### PR TITLE
No-Issue: Set BMH annotation for master role

### PR DESCRIPTION
**Pull request by pawanpinjarkar**
# Assisted Pull Request

## List all the issues related to this PR

- [ ] New Feature
- [ ] Bug fix
- [ ] Tests
- [ ] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [ ] Automation (CI, tools, etc)
- [ ] Cloud
- [ ] Operator Managed Deployments
- [x] None

## How was this code tested?

- [ ] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [ ] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [x] No tests needed

## Assignees

/cc @
/cc @

## Checklist

- [ ] Title and description added to both, commit and PR.
- [ ] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [ ] Reviewers have been listed
- [ ] This change does not require a documentation update (docstring, `docs`, README, etc)
- [ ] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- [ ] Are the title and description (in both PR and commit) meaningful and clear?
- [ ] Is there a bug required (and linked) for this change?
- [ ] Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

**Comment by osherdp** [comment]
@pawanpinjarkar any context for the change?
just to understand how it meets your work and why automatic role-assignment is not good enough :)

File: deploy/operator/ztp/baremetalHost.j2 [hunk 1]
```diff
@@ -20,6 +20,7 @@ metadata:
     infraenvs.agent-install.openshift.io: '{{ infraenv_name }}'
   annotations:
     bmac.agent-install.openshift.io/hostname: '{{ host["name"] }}'
+    bmac.agent-install.openshift.io/role: master
```
Comment (YuviGold) [inline deploy/operator/ztp/baremetalHost.j2:23]: What about workers?

**Comment by pawanpinjarkar** [comment]
@osherdp @YuviGold My bad, the auto assignment does work. This change is not needed.

**Pull request closed without merging**